pkg/bff/server.go

```diff
@@ -196,6 +196,7 @@ func (s *Server) setupRoutes() (err error) {
 		// GDS public routes (no authentication required)
 		v1.GET("/lookup", s.Lookup)
 		v1.POST("/register/:network", s.Register)
+		v1.GET("/verify-contact", s.VerifyContact)
```

Comment:
Endpoint matches the current URL - happy to change to `/verify` if that seems better.

Reply:
`/verify` does seem a bit nicer; though if the ship has sailed already and we're comfortable with `/verify-contact` I won't make a fuss